pkg/kudoctl/cmd/package_list_plans.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"sort"
+
+	"github.com/gosuri/uitable"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+)
+
+type packageListPlansCmd struct {
+	fs             afero.Fs
+	out            io.Writer
+	path           string
+	RepoName       string
+	PackageVersion string
+}
+
+const (
+	packageListPlansExample = `# show plans from local-folder (where local-folder is a folder in the current directory)
+  kubectl kudo package list plans local-folder
+
+  # show plans from zookeeper (where zookeeper is name of package in KUDO repository)
+  kubectl kudo package list plans zookeeper`
+)
+
+func newPackageListPlansCmd(fs afero.Fs, out io.Writer) *cobra.Command {
+	list := &packageListPlansCmd{fs: fs, out: out}
+
+	cmd := &cobra.Command{
+		Use:     "plans [operator]",
+		Short:   "List operator plans",
+		Example: packageListPlansExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := validateOperatorArg(args); err != nil {
+				return err
+			}
+			list.path = args[0]
```

Comment:
should this really be named path since it's actually path or package name or url?

Reply:
yeah... this one is really challenging to name... suggestions?